bioptim/interfaces/acados_interface.py

```diff
@@ -238,7 +347,7 @@ def __set_costs(self, ocp):
             self.acados_ocp.cost.yref_e = np.zeros((self.acados_ocp.cost.W_e.shape[0],))
 
         elif self.acados_ocp.cost.cost_type == "EXTERNAL":
-            raise RuntimeError("External is not interfaced yet, please use NONLINEAR_LS")
+            raise RuntimeError("EXTERNAL is not interfaced yet, please use NONLINEAR_LS")
```

Comment:
This could be an enum and not a text (I don't mean in self.acados_ocp.cost.cost_type which necessitates text, but another variable and the option (solver_options["cost_type"]). Would that make sense?